app/models/metasploit/cache/license.rb

```diff
@@ -0,0 +1,63 @@
+# Represents licenses like BSD, MIT, etc used to provide license information for {Metasploit::Cache::Module::Instance modules}
```

Comment:
Don't link to `Metasploit::Cache::Module::Instance` here as it is part of the old schema's classes and will be removed once we finish all the new schema work.  Instead due no link and use the text `Metasploit modules` or link to each type of Metasploit Module individually since there's no shared base class for instances in the new schema.